### PR TITLE
Generate static metadata

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**

### CODE REVIEW

**Comment by Emmanuel-Develops** [comment]
closes #146

**Comment by 0tuedon** [comment]
@Emmanuel-Develops  once this is merged this means i have to restructure the file directory PR again?

File: .github/workflows/generate-static-data.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+
+    - name: Install yarn
+      run: npm i yarn -g
+
+    - name: Intall dependencies
```
Comment (Extheoisah) [inline .github/workflows/generate-static-data.yml:20]:
```suggestion
    - name: Install dependencies
```

**Comment by Emmanuel-Develops** [comment]
> @Emmanuel-Develops once this is merged this means i have to restructure the file directory PR again?

yes. Let me add a commit with the generated metadata so you can see the structure

**Comment by Extheoisah** [comment]
You need to update the branch. Github says its out of date

**Review by 0tuedon — APPROVED** [review]
I guess you have to update the branch

**Pull request merged**